### PR TITLE
INT b 20465 customer selecting counseling office

### PR DESCRIPTION
**Pull request by JonSpight**
## Summary
When a customer selects an origin duty location assigned to government counseling, then they will also be asked to select a counseling office within the same GBLOC. This field will be called "Counseling Office".

### How to test as a Customer
1.  Login as a Customer
2. Select a counseling Office while creating a new move
3. Check the Moves data table for corresponding counseling_office_id

### How to test as a Service Counselor
1.  Login as a Service Counselor
2. Verify the counseling queue's counseling Office column matches what was created in previous move

## 508 Screenshots
[508Screenshots-20465.pdf](https://github.com/user-attachments/files/16971315/508Screenshots-20465.pdf)

### CODE REVIEW

**Review by loganwc — CHANGES_REQUESTED** [review]

**Comment by loganwc** [comment]
You have a few files in there that I don't think are apart of your PR

File: .project [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (JonSpight) [reply .project]: removed it.

[the remark this answers is not in the document]
Comment (JonSpight) [reply .project]: along with the other files .

**Comment by pambecker** [comment]
I'm not seeing the new text required on the UI to meet the first AC in the story.

Given I am a Milmove customer
When I am on the create orders page
Then I see this instructional text below the Current duty Location field:
"Select an origin duty location that most closely represents your current physical location, not where your shipment will originate, if different. This will allow a nearby transportation office to assist you."

image

**Comment by pambecker** [comment]
We need to have the 'Required' label on this new field and not enable the Next button until it's filled out.  This is how the other fields work.  
And the Red Required text when the field is cleared out like the other fields do.

image

image

**Comment by pambecker** [comment]
Found an issue.  When I first pick a govt counseled duty location and choose a counseling office, then go back and change the origin duty location to a GHC prime counseled (NS Mayport), complete the form and save.  My previously picked counseling office is still saved to the counseling_transportation_office_id in the DB and it should have been nulled out.

**Review by pambecker — CHANGES_REQUESTED** [review]
Added some feedback for cleanup.

**Pull request closed without merging**